docs/release-notes/checkpoint-download-through-master.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  API: Support downloading checkpoints through the master. It only supports AWS S3 for now and relies on implicit IAM permissions on the master. Two endpoints are added for downloading checkpoints in different file formats:
```

Comment:
Can we leave the release note out?  Generally we discourage users from using the REST API directly, since it has a habit of breaking changes frequently.

If we don't have specific customers asking for this feature, I don't think we should announce it.

Reply:
Sure. I had no idea.